extensions/src/main/java/org/pentaho/platform/plugin/services/cache/CacheManager.java

```diff
@@ -400,7 +400,10 @@ public Set getAllEntriesFromRegionCache( String region ) {
   public void removeFromRegionCache( String region, Object key ) {
     if ( checkRegionEnabled( region ) ) {
       HvCache hvcache = (HvCache) regionCache.get( region );
-      hvcache.evictEntityData( (String) key );
+      try ( SessionImpl session = (SessionImpl) hvcache.getSessionFactory().openSession() ) {
```

Comment:
shouldn't be a `catch` statement statement here and at least a log line here, instead of shallowing the error.
The log level can be [error, debug, trace] just people can debug it in the field without a code change.